util/module.h

```diff
@@ -306,6 +306,17 @@ typedef int inplace_cb_edns_back_parsed_func_type(struct module_qstate* qstate,
 typedef int inplace_cb_query_response_func_type(struct module_qstate* qstate,
 	struct dns_msg* response, int id, void* cb_args);
 
+/**
+ * Function called when looking for (expired) cached answers during the serve
+ * expired logic.
+ * Called as func(qstate, lookup_qinfo)
+ * Where:
+ *	qstate: the query state.
+ *	lookup_qinfo: the qinfo to lookup for.
+ */
+typedef struct dns_msg* serve_expired_lookup_func_type(
```

Comment:
It would be nice to have a fptr_wlist item for this func_type to check for the possible allowed values.

Reply:
Fixed in https://github.com/NLnetLabs/unbound/pull/159/commits/ea29698eebcfae31874d04f7e8d9564591d52dd2